src/components/Indicator.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import {StyleSheet, View} from 'react-native';
+import {OnyxCollection, withOnyx} from 'react-native-onyx';
+import {OnyxEntry} from 'react-native-onyx/lib/types';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import * as UserUtils from '@libs/UserUtils';
+import useTheme from '@styles/themes/useTheme';
+import useThemeStyles from '@styles/useThemeStyles';
+import * as PaymentMethods from '@userActions/PaymentMethods';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {BankAccountList, FundList, LoginList, Policy, PolicyMembers, ReimbursementAccount, UserWallet, WalletTerms} from '@src/types/onyx';
+
+type CheckingMethod = () => boolean;
+
+type IndicatorOnyxProps = {
+    /** The employee list of all policies (coming from Onyx) */
+    allPolicyMembers: OnyxCollection<PolicyMembers>;
+
+    /** All the user's policies (from Onyx via withFullPolicy) */
+    policies: OnyxCollection<Policy>;
+
+    /** List of bank accounts */
+    bankAccountList: OnyxEntry<BankAccountList>;
+
+    /** List of user cards */
+    fundList: OnyxEntry<FundList>;
+
+    /** The user's wallet (coming from Onyx) */
+    userWallet: OnyxEntry<UserWallet>;
+
+    /** Bank account attached to free plan */
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+
+    /** Information about the user accepting the terms for payments */
+    walletTerms: OnyxEntry<WalletTerms>;
+
+    /** Login list for the user that is signed in */
+    loginList: OnyxEntry<LoginList>;
+};
+
+function Indicator({
```

Comment:
```suggestion
};

type IndicatorProps = IndicatorOnyxProps;

function Indicator({
```

NAB: Just define the `IndicatorProps`, in case someone in the future need to add some prop to the component ;)